File: app/src/main/java/com/example/agroapp/model/HomepageData.java
package com.example.agroapp.model;

public class HomepageData {

    private BannerData bannerData;
    private ArticleList articleList;

    public BannerData getBannerData() {
        return bannerData;
    }

    public void setBannerData(BannerData bannerData) {
        this.bannerData = bannerData;
    }

    public ArticleList getArticleList() {
        return articleList;
    }

    public void setArticleList(ArticleList articleList) {
        this.articleList = articleList;
    }
}

File: kotlin/src/main/java/com/example/kotlin/data/HomepageData.kt
package com.example.kotlin.data

/**
 *  author : jiangxue
 *  date : 2023/6/2 10:23
 *  description :
 */
class HomepageData {
    private var bannerData: BannerData? = null
    private var articleList: ArticleList? = null
    fun getBannerData(): BannerData? {
        return bannerData
    }

    fun setBannerData(bannerData: BannerData?) {
        this.bannerData = bannerData
    }

    fun getArticleList(): ArticleList? {
        return articleList
    }

    fun setArticleList(articleList: ArticleList?) {
        this.articleList = articleList
    }
}